modules/graylog/collector.go

```diff
@@ -0,0 +1,59 @@
+package graylog
+
+import (
+	"github.com/NETWAYS/support-collector/pkg/collection"
+	"github.com/NETWAYS/support-collector/pkg/obfuscate"
+	"os"
+	"path/filepath"
+)
+
+const ModuleName = "graylog"
+
+var relevantPaths = []string{
+	"/etc/graylog",
+}
+
+var possibleDaemons = []string{
+	"/usr/lib/systemd/system/graylog-server.service",
+}
+
+var files = []string{
+	"/etc/graylog",
+}
+
+var obfuscators = []*obfuscate.Obfuscator{
+	obfuscate.NewFile(`(?i)(?:password_secret|root_password_sha2|elasticsearch_hosts|mongodb_uri).*\s*=\s*(.*)`, `conf`),
+}
+
+func Detect() bool {
+	for _, path := range relevantPaths {
+		_, err := os.Stat(path)
+		if err == nil {
+			return true
+		}
+	}
+
+	return false
+}
+
+func Collect(c *collection.Collection) {
+	if !Detect() {
+		c.Log.Info("Could not find Graylog")
+		return
+	}
+
+	c.Log.Info("Collecting Graylog information")
+
+	c.RegisterObfuscators(obfuscators...)
+
+	c.AddInstalledPackagesRaw(filepath.Join(ModuleName, "packages.txt"), "*graylog*", "*elasticsearch*", "*opensearch*", "*mongo*") //nolint:lll
```

Comment:
For the "external tools" extra modules should be built and the packages for them should be collected in them instead of here. 
Should happen before the next release, which would not explain the packages in this module.

Reply:
I just thought it might be nice to have this information when only collecting Graylog, since they are always part of the stack.

Reply:
In case we want to get a overview from the whole stack, all modules from the stack should be collected.
In that case we would have duplicates in the modules

Reply:
:+1: Done